app/coffee/Features/Announcements/AnnouncementsHandler.coffee

```diff
@@ -48,7 +51,9 @@ module.exports = AnnouncementsHandler =
 				announcement.id == lastSeenBlogId
 
 			announcements = _.map announcements, (announcement, index)->
-				if announcementIndex == -1
+				if announcement.date < userSignupDate
```

Comment:
Is announcement.date always midnight of that day? so a user signing up at any time on the day of the announcement won't see it? Just checking it isn't the other way around, and user signing up on the same day as the annoucement will always see it due to the 24 hour window of a day.